### DISCUSSION TITLE
Gazebo Multicopter motor model plugin and controller

### DISCUSSION THREAD

**Discussion by omr-web** [bitcraze/discussions#566] [Q&A]
hello everyone,

I am trying to make an altitude and attitude controller using the multicopter motor model plugin. There is an interesting situation that I noticed. When I give the drone a constant velocity, the drone starts to rise, but after a certain height, although the velocity value increases, it starts to descend and descends to the ground. when I give a higher constant velocity value, it goes up to a higher height and descends again. I think of this situation as a feature of the gazebo world. What is the reason for this situation? How should my controller behave towards this situation?

thanks your helps.

**Comment by krichardsson** [comment]
Duplicate of #568 (actually the other way around, but #568 contains more information)
